### PR TITLE
feat: added rightContent option on large checkboxes

### PR DESCRIPTION
**Pull request by berglindoma13**
# ...

Attach a link to issue if relevant

## What

Added an option of right content on large checkbox boxes

## Why

Needed for new citizenship application where tags need to be displayed on checkboxes

## Screenshots / Gifs

After

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx [hunk 1]
```diff
@@ -16,8 +16,12 @@ import { useLocale } from '@island.is/localization'
 
 import { getDefaultValue } from '../../getDefaultValue'
 
+interface CheckboxFieldPropsExtra extends CheckboxField {
+  rightContent?: React.ReactNode
+}
+
 interface Props extends FieldBaseProps {
-  field: CheckboxField
+  field: CheckboxFieldPropsExtra
```
Comment (Thorrik58) [inline libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx:24]: This shouldnt be the way to solve this, you should add this prop to the CheckBoxField in @island.is/application/types.

File: libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx [hunk 2]
```diff
@@ -69,6 +74,7 @@ export const CheckboxFormField: FC<Props> = ({
           onSelect={onSelect}
           split={width === 'half' ? '1/2' : '1/1'}
           backgroundColor={backgroundColor}
+          rightContent={rightContent}
```
Comment (Thorrik58) [inline libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx:77]: Was this change to the application system approved by design and added to the design system?
Also if this is a change to the field you should update the checkboxformfield story in storybook to demo the changes.

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Just minor comments, need to move the type change and the prop to the checkboxformfield story. Also some questions regarding the design process.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR closed its doors. Sat around for a day after the stale tag, then reckoned it's not worth the fuss.

**Pull request closed without merging**